models/report.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (sequelize, DataTypes) => {
+  const Report = sequelize.define('Report', {
+    report: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    articleId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+
+    },
+  });
+    // Report.associate = models => Report.hasOne(models.Profile, {
```

Comment:
Why are these parts commented?

Reply:
> Why are these parts commented?

The comment was a mistake, I would remove them.

Reply:
Okay. You can add those relationships in the already existing related files then. 

Also, I'd like you to help change the structure of our migration files naming.

replace the numerical parts with `001`, `002` and so on till you get to the report migration file which will be `006`

Reply:
I've made requested changes and added the relationships.

Reply:
Okay.

The commented code is still in the report model however.

Reply:
It seems the `Reports` table is not being created when migrations are done.

Reply:
This was a test related thing. Disregard.

> It seems the `Reports` table is not being created when migrations are done.

This was a test-related thing. Disregard.

Reply:
The `Reports` table is created when migrations are done.
I traced the travis error to the relationship i defined in the user model.
`User.hasMany(models.Report, {`
      `foreignKey: 'userId',`
      `onDelete: 'CASCADE'`
    });`